src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -102,10 +103,11 @@ export const MyFieldAndBackground = () => {
       <Formik initialValues={initialValues} onSubmit={(values) => updatePerson.mutate(values)} enableReinitialize>
         {({ isSubmitting, dirty, setFieldValue, resetForm }: FormikProps<PersonBackgroundFormData>) => (
           <Form>
-            <Box sx={{ mx: '1rem', mt: '1rem', maxWidth: '60rem' }}>
-              <div>
-                <Typography variant="h2">{t('my_page.my_profile.field_and_background.field')}</Typography>
-                <Typography variant="h3" sx={{ mb: '1rem', mt: '1.5rem' }}>
+            <Box sx={{ display: 'flex', flexDirection: 'column', m: '1rem', maxWidth: '60rem', gap: '1rem' }}>
+              <Typography variant="h2">{t('my_page.my_profile.field_and_background.field_and_background')}</Typography>
+              <ProfileBox>
+                <Typography variant="h3">{t('my_page.my_profile.field_and_background.field')}</Typography>
+                <Typography sx={{ mb: '1.5rem', mt: '0.5rem' }}>
```

Comment:
Jeg liker å bruke de innebygde `gutterBottom` og `paragraph` hvor de passer, sånn at vi får en ganske lik spacing overalt. Eller blir det fort mye inkonsistens, med ulik spacing på alt mellom typisk `0.2rem` og `2rem` rundt omkring.
Men det er et motarbument at `gutterBottom` og `paragraph` abstraherer bort hva spacingen faktisk er da, så ingen deal-breaker å bruke faktiske verdier heller, men man må kanksje være litt mer bevisst på hvilke verdier man setter.

```suggestion
                <Typography variant="h3" gutterBottom>{t('my_page.my_profile.field_and_background.field')}</Typography>
                <Typography paragraph>
```